plugin-server/src/worker/ingestion/event-pipeline/1-emitToBufferStep.ts

```diff
@@ -36,7 +36,10 @@ export async function emitToBufferStep(
         // use the old logic.
         // TODO: If we want to enable this for all teams, we can remove this
         // check.
-        if (runner.hub.conversionBufferTopicEnabledTeams.has(event.team_id)) {
+        if (
```

Comment:
@hazzadous could we write a unit test for the changed behavior?

This file was previously fully covered, now not so anymore.